src/web/mod.rs

```diff
@@ -15,11 +15,11 @@ macro_rules! ctry {
             Err(error) => {
                 let request: &::iron::Request = $req;
 
+                // TODO this should be `capture_anyhow` if error is `anyhow::Error`
```

Comment:
Let me think about how we could do this, I'll play around with it :)

Reply:
https://github.com/rust-lang/docs.rs/commit/d6ea893a7829bfb75a1c90e91633641101ee1e65 should make it work once #1489 is merged :)

Reply:
And it would be really cool if you could change all the current `log::error!` calls to use `web::report_error` instead - do you mind doing that? Feel free to move that function around and change the interface, I just want to keep the original backtrace :)

Reply:
@jyn514 good point here, 

I'll dig into generalizing the reporting into one place that we then use everywhere.